api/Makefile

```diff
@@ -167,7 +167,7 @@ db-migrate-heads: ## Show migrations marked as a head
 	$(alembic_cmd) heads $(args)
 
 db-seed-local:
-	$(PY_RUN_CMD) db-seed-local
+	$(PY_RUN_CMD) db-seed-local $(args)
```

Comment:
👍

Reply:
(chore): update the docs to include iterations: https://github.com/HHS/simpler-grants-gov/blob/chouinar/1455-query-box/documentation/api/database/database-local-usage.md?plain=1#L28